yt_dlp/extractor/iwara.py

```diff
@@ -73,8 +78,17 @@ def _real_extract(self, url):
             r'<title>([^<]+)</title>', webpage, 'title'), ' | Iwara')
 
         thumbnail = self._html_search_regex(
-            r'<video[^>]+id=[\'"]video-player[\'"][^>]+poster=[\'"]([^\'"]+)',
-            webpage, 'thumbnail', default=None)
+            r'poster=[\'"]([^\'"]+)', webpage, 'thumbnail', default=None)
+
+        uploader = self._html_search_regex(
+            r'class="username">([^<]+)', webpage, 'uploader', fatal=False)
+
+        upload_date = self._html_search_regex(
+            r'作成日:([^\s]+)', webpage, 'upload_date', fatal=False).replace("-", "")
```

Comment:
```suggestion
        upload_date = unified_strdate(self._html_search_regex(
            r'作成日:([^\s]+)', webpage, 'upload_date', fatal=False))
```

(and add import from utils)